src/runtime/nitro/plugins/02a-preprocessHtml.ts

```diff
@@ -10,21 +10,6 @@ export default defineNitroPlugin((nitroApp) => {
     if (!security?.sri && (!security?.headers || !security?.headers.contentSecurityPolicy)) {
       return
     }
-
-    type Section = 'body' | 'bodyAppend' | 'bodyPrepend' | 'head'
-    const sections = ['body', 'bodyAppend', 'bodyPrepend', 'head'] as Section[]
-    const cheerios = {} as Record<Section, ReturnType<typeof cheerio.load>[]>
-    for (const section of sections) {
-      cheerios[section] = html[section].map(element => {
-        return cheerio.load(element, {
-          xml: {
-            // Disable `xmlMode` to parse HTML with htmlparser2.
-            xmlMode: false,
-            decodeEntities: false
-          },
-        }, false)
-      })
-    }
-    event.context.cheerios = cheerios
+    event.context.cheerios = html
```

Comment:
@Baroshem I'm not entirely sure, should this be trimmed? Is there anything that needs to be done with each cheerio? Or is it meant to be as is?

Reply:
I think @vejja would be a best person to answer here as he was implementing the cheerio :)

Reply:
But I think that the best case would be to get rid of all library in general and use regexes instead for better performance :)

Reply:
> But I think that the best case would be to get rid of all library in general and use regexes instead for better performance :)

I apologize, it appears I misspoke. I meant to ask, what should each html section receive? I referred to a section as cheerio because the context is cheerios, I apologize for the misunderstanding.